e2e/cypress/integration/messaging/input_box_expands_spec.js

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [number] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+import * as TIMEOUTS from '../../fixtures/timeouts';
+
+describe('Messaging', () => {
+    before(() => {
+        // # Make sure the viewport is the expected one, so written lines always create new lines
+        cy.viewport(1000, 660);
```

Comment:
Just curious about this comment, which viewport does not create new line per test step?

Also, the default viewport being used by Cypress works fine to me.

Reply:
I understand that if we change the size of the viewport (for example, if for any reason we change the default viewport size) the number of lines needed to have the full height may be different. Therefore, that is why I set the viewport size. Calculating on the run the number of lines needed may be also troublesome, but it is another possibility.

Reply:
Got it.  However, I suggest not to add it and have the test run against the default viewport we set in Cypress.